content/blog/2018/02/2018-02-19-gsoc2018-announcement.adoc

```diff
@@ -0,0 +1,64 @@
+---
+:layout: post
+:title: "Jenkins is accepted to Google Summer Of Code 2018, join us!"
+:tags:
+- gsoc
+- gsoc2018
+- events
+- community
+:author: oleg_nenashev
+---
+
+image:/images/gsoc/jenkins-gsoc-logo_small.png[Jenkins GSoC, role=center, float=right]
+
+We are happy to announce that Jenkins project has been accepted to
+link:https://summerofcode.withgoogle.com/[Google Summer of Code 2018].
+This year we invite students and mentors to join the Jenkins community and work together
+on various initiatives: core, plugins, development tools and infrastructure.
+
+Our mentors have already created some link:/projects/gsoc/gsoc2018-project-ideas[project ideas].
+For example, you are welcome to work on the new link:/projects/gsoc/gsoc2018-project-ideas/#jenkins-configuration-as-code[Configuration-as-Code initiative] or
+to help link:/projects/gsoc/gsoc2018-project-ideas/#code-coverage-api-plugin[creating standard API for Code Coverage plugins].
+If you like test automation, there is a proposal to
+link:/projects/gsoc/gsoc2018-project-ideas/#improvements-to-the-jenkins-acceptance-test-harness[Jenkins Acceptance Test Harness].
+Over the next weeks we plan to extend this list by new even more project ideas.
+
+All information about the Jenkins GSoC is available on its link:/projects/gsoc/[subproject page].
+
+== I am a student. How to apply?
```

Comment:
This title and the section for it have some "inconsistency in perspective", going from me/you to "impersonal".

E.g. after *In am a student*, I'd expect *How do I apply?* This seems to continue below.